docs/source/policies/security_response_policy.rst

```diff
@@ -0,0 +1,28 @@
+NI LinuxRT Security Response Policy
+===================================
+
+NI is committed to resolving major security vulnerabilities discovered in the x64 NI LinuxRT distribution images and core package feeds.
+
+The LinuxRT development team regularly monitors the `NIST National Vulnerability Database <https://nvd.nist.gov/>`_ for published vulnerabilities, and uses that information to audit the x64 NILRT distribution software. When a vulnerability is discovered, the team evaluates its impact to our products and our customers' supported workflows, and assigns the vulnerability a vendor severity score. In many cases, NI's vendor score agrees with the NIST-assigned score. But the team may downgrade a vulnerability if it does not affect the system in a supported configuration.
```

Comment:
For ease of reviewing, I'd suggest we avoid putting paragraphs entirely on one line. The maximum line length is inconsistent throughout this repo--perhaps I'll fix that at some point--but pick something reasonable for this PR at least (I don't really care about the exact number; I'd probably choose one of 72, 80, 100, or 120 columns).

Reply:
Why does it matter? A 120 character line would still flag the whole line as being different if a word is changed. And differs which highlight intermediate changes like that will work the same.

Reply:
Suggestion blocks become a minor annoyance due to the large copy-paste and sideways scrolling. I don't feel too strongly about it though.

```suggestion
The LinuxRT development team regularly monitors the `NIST National Vulnerability Database <https://nvd.nist.gov/>`_ for published vulnerabilities, and uses that information to audit the x64 NILRT distribution software. When a vulnerability is discovered, the team evaluates its impact to our products and our customers' supported workflows, and assigns the vulnerability a vendor severity score. In many cases, NI's vendor score agrees with the NIST-assigned score, but the team may downgrade a vulnerability if it does not affect the system in a supported configuration.
```